sdk/src/Core/Amazon.Runtime/Credentials/DefaultAWSTokenIdentityResolver.cs

```diff
@@ -26,10 +26,8 @@ public class DefaultAWSTokenIdentityResolver : IIdentityResolver<AWSToken>
     {
         private readonly IAWSTokenProvider _tokenProvider;
 
-        public DefaultAWSTokenIdentityResolver() : this(null) { }
-
-        public DefaultAWSTokenIdentityResolver(string profileName)
```

Comment:
This constructor was redundant, the `ProfileTokenProvider` already searches for the profile name in the correct places (config file, environment variable, etc...): https://github.com/aws/aws-sdk-net/blob/63cb4e09f3f21b33a37d2f79cc79744c9a9feef8/sdk/src/Core/Amazon.Runtime/_bcl%2Bnetstandard/Tokens/ProfileTokenProvider.cs#L102-L105